safe-global/safe-apps-sdk
# Feat: Add safe test app to the repo

This PR:
- Partly implements https://github.com/safe-global/safe-react-apps/issues/473 by moving the original test app from https://github.com/gnosis/safe-test-app to the safe-apps-sdk repo. I couldn't make the vite app work in a monorepo, so I think we should either deprecate it or keep it as a separate repo.
- Updates all dependencies 💪 

How to test:
1. checkout to the branch
2. run `yarn workspace @gnosis.pm/safe-apps-test-app start`
3. Add a custom safe app to the test interface and check if it works
